requirements/requirements.txt

```diff
@@ -3,15 +3,15 @@ Cython==0.17.4
 restkit==4.2.0
 jsonobject-couchdbkit==0.6.5.5
 jsonobject==0.4.3
-celery==3.0.24
+celery==3.1.11
 decorator==3.3.2
 django==1.5.6
 django-braces==1.3.1
-django-celery==3.0.23
+django-celery==3.1.10
 django-crispy-forms==1.4.0
 django-json-field==0.5.5
-kombu==2.5.16
-billiard==2.7.3.32
+kombu==3.0.16
```

Comment:
They didn't update automatically when I upgraded celery, and I remember having a hard time finding the right versions of everything to make it work together, so pinning the right version is definitely the right thing to do.